src/Nav.php

```diff
@@ -84,6 +84,10 @@ class Nav extends Widget
     /**
      * @var bool whether to activate parent menu items when one of the corresponding child menu items is active.
      */
+    public $activateContainer = true;
+    /**
+     * @var bool whether to activate the container when the menu item is active.
```

Comment:
Seems descriptions aren't for their corresponding properties.

Reply:
My bad, fixed now